packages/terra-worklist-data-grid/src/proptypes/cellShape.js

```diff
@@ -4,7 +4,7 @@ const cellShape = PropTypes.shape({
   /**
    * String indicating content to be displayed for the cell.
    */
-  content: PropTypes.string,
+  content: PropTypes.node,
```

Comment:
Why was this changed to a node type?  Should it actually be an element prop type so that consumers pass the React component to be rendered?

Reply:
`node` supports react components and strings. Depending on the requirements, `node` would work if we want consumers to be able to pass in a string as the cell value or a react component for more complex content.